JavadocStyleGuide.md

```diff
@@ -0,0 +1,242 @@
+ * Use the {@code Project} instance to configure the project.
+ */
+```
+
+### 1.1.7 Block tags
+
+Any of the standard "block tags" that are used appear in the order `@param`, `@return`, `@throws`, `@deprecated`, and these four types never appear with an empty description:
+
+```java
+/**
+ * @param  url  an absolute URL giving the base location of the image
+ * @param  name the location of the image, relative to the url argument
+ * @return      the image at the specified URL
+ * @see         Image
+ */
+```
+
+Full list of tags (in order):
+
+| # | Tag           | Usage                                                                          | Notes                                                                                      |
+|---|---------------|--------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------|
+| 1 | `@param`      | Methods and constructors only                                                  | Include this if applicable                                                                 |
+| 2 | `@return`     | Methods only                                                                   | Include this if applicable                                                                 |
+| 3 | `@throws`  | Same as `@exception`                                                              | Include this if applicable                                                                 |
+| 4 | `@see`        | Adds a “See Also” heading with a link or text entry that points to a reference | `@see string`<br>`@see <a href=”URL#value”>label</a>`<br>`@see package.class#member label` |
+| 5 | `@since`      | Adds a “Since” heading                                                         | Include the Gradle version if applicable                                                   |
+| 6 | `@deprecated` | Adds a comment indicating that this API should no longer be used               | Make sure to have an alternative API linked                                                |
+
+Custom Gradle tags:
+
+| # | Tag         | Usage                                        | Notes                                                                                                                                                                                                               |
+|---|-------------|----------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1 | `@apiSpec`  | Adds a “API Requirements” heading            | A description that applies equally to all valid implementations of the method, including preconditions, postconditions, etc.                                                                                        |
```

Comment:
These spec/note tags are to appear below `@deprecated`, correct?  They're not Custom Gradle tags (though they are less frequently needed) and should probably just be part of a single list.

Reply:
These became standard in the newer JDK but I don't think they are standard in JDK 11. I will update accordingly either way.